src/cascadia/TerminalSettingsModel/CascadiaSettingsSerialization.cpp

```diff
@@ -893,6 +893,20 @@ try
         settings->_hash = _calculateHash(settingsString, lastWriteTime);
     }
 
+    // GH#13936: We're interested in how many users opt out of useAtlasEngine,
+    // indicating major issues that would require us to disable it by default again.
+    {
+        const auto& globals = settings->_globals;
+        TraceLoggingWrite(
+            g_hSettingsModelProvider,
+            "AtlasEngine_Enabled",
+            TraceLoggingDescription("Event emitted upon settings load, containing whether one is opted-in/out of useAtlasEngine"),
+            TraceLoggingBool(globals->HasUseAtlasEngine(), "UseAtlasEngineOverridden", "true if useAtlasEngine is set"),
+            TraceLoggingBool(globals->UseAtlasEngine(), "UseAtlasEngine", "true if AtlasEngine is enabled"),
+            TraceLoggingKeyword(MICROSOFT_KEYWORD_MEASURES),
+            TelemetryPrivacyDataTag(PDT_ProductAndServiceUsage));

```

Comment:
In team sync you mentioned a bunch of performance data we wanted to get insights into as well. Atlas resizing, etc.
Did you want to include those in this PR as well?

Reply:
I'd have to figure out how to implement them first and I felt like getting this crucial change in ASAP is much more important than that.

Reply:
I agree with Leonard on this one. Let's start with simple